controller/work_item_children_blackbox_test.go

```diff
@@ -155,7 +155,7 @@ func (s *workItemChildSuite) SetupTest() {
 	require.Nil(s.T(), err)
 	s.T().Logf("Created bug2 with ID: %s\n", *bug2.Data.ID)
 
-	bug3Payload := CreateWorkItem(s.userSpaceID, workitem.SystemBug, "bug3")
+	bug3Payload := getCreateWorkItemPayload(s.userSpaceID, workitem.SystemBug, "bug3")
```

Comment:
Oh yes, I'm fine with using such simpler, more idiomatic function names, but the point is, this function return the payload for the 'Create WorkItem' operation.